drop-boxes/register-mtb-data-dropbox/register-mtb-data-dropbox.py

```diff
@@ -83,7 +83,7 @@
 DATA_AVAILABLE_JSON = tracking_helper.get_data_available_status_json()
 
 ### We need this object to update the sample status later
-SAMPLE_TRACKER = SampleTracker.createLocationIndependentSampleTracker(SERVICE_REGISTRY_URL, SERVICE_CREDENTIALS, DATA_AVAILABLE_JSON)
+SAMPLE_TRACKER = SampleTracker.createLocationIndependentSampleTracker(SERVICE_REGISTRY_URL, SERVICE_CREDENTIALS)
```

Comment:
not used? defined in 134 again

Reply:
it's used, but I will remove the duplicate definition